hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java

```diff
@@ -34,21 +32,38 @@
 public class RSGroupInfo {
   public static final String DEFAULT_GROUP = "default";
   public static final String NAMESPACE_DESC_PROP_GROUP = "hbase.rsgroup.name";
+  public static final String TABLE_DESC_PROP_GROUP = "hbase.rsgroup.name";
```

Comment:
Its intentional this and the define above have same value?

Reply:
Yes, intentional. We can use the same name in namespace and table properties, the only problem is that the old name contains a 'NAMESPACE' and it is IA.Public, so we need to introduce a new 'TABLE' one.